src/code/code_800EC960.c

```diff
@@ -3774,41 +3785,41 @@ void func_800F6828(u8 arg0) {
     }
 }
 
-void func_800F68BC(s8 arg0) {
-    D_8013060C = arg0;
+void Audio_SetDemoActive(s8 active) {
+    sAudioDemoActive = active;
 }
 
-void func_800F68D4(u16 arg0, Vec3f* arg1, u8 arg2, u32* arg3, u32* arg4, u32* arg5) {
-    if (D_8013060C == 0) {
-        Audio_PlaySoundGeneral(arg0, arg1, arg2, arg3, arg4, arg5);
+void Audio_PlaySoundGeneralIfNotDemo(u16 sfxId, Vec3f* pos, u8 arg2, f32* freqScale, f32* arg4, s8* reverbAdd) {
+    if (!sAudioDemoActive) {
```

Comment:
think the name is a bit verbose, but we can look into improving it at a later time.
Am curious on your thoughts of `PlaySoundGeneral` in general though. this function was named super early on with almost no knowledge of the audio system afaik. So Im curious if we have a better name for it

Reply:
mainly wondering things like: does `Audio_PlaySfx` make sense? is the word general needed? if playsfx is too broad, is there a way to make it a bit more specific other than saying the word "general"?
stuff like that

Reply:
I think `PlaySoundGeneral` is a decent name (I read the "General" as "can play any sfxId, with any sound settings"), but what we should do is add some macros for `Audio_PlaySoundGeneral(sfxId, pos, 4, &D_801333E0, &D_801333E0, &D_801333E8)` (playing sound with default pitch and reverb) and `Audio_PlaySoundGeneral(sfxId, &D_801333D4, 4, &D_801333E0, &D_801333E0, &D_801333E8)` (default source position). (The `4` in those calls doesn't mean anything btw, the parameter is completely ignored afaict.)

The naming of sfx vs sound is a good question. I think "sfx" *might* mean something else internally within audio, like a separate kind of instrument from "program"/"drum"? But I don't know too much about that and whether it could be confusing. Certainly given the name "sfxId" it would be good to unify terminology here...

Reply:
Alright cool. yea as we get further along with audio docs I think we'll be able to unify things better. just something to keep in the back of our minds as we keep documenting stuff